src/Symfony/Component/DependencyInjection/ExpressionLanguage.php

```diff
@@ -14,6 +14,10 @@
 use Psr\Cache\CacheItemPoolInterface;
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage as BaseExpressionLanguage;
 
+if (!class_exists(BaseExpressionLanguage::class)) {
+    throw new \ReflectionException(BaseExpressionLanguage::class.' not found.');
```

Comment:
related to #32995

Reply:
How does that solve #32995 ? It‘s still going to be an exception thrown during autoload. Thus we should get a fatal here, if someone tries to autoload the `ExpressionLanguage` class defined in _this_ file.

Reply:
#32995 is about the fact that PHP 7.4 does not properly allows us to turn a missing parent class into a catcheable exception.
This is throwing an exception *before* PHP encounters the missing parent class.

Reply:
The issue described in #32995 happens when the parent class fails to be loaded *while the engine already registered the child as "now loading". This `throw` happens *before* the child is flagged as such, thus the exception can be properly caught.

Reply:
Note that a `return;` would do it too. I'm going to update.